scripts/build

```diff
@@ -39,12 +38,35 @@ esac
 
 OS="`uname`"
 case $OS in
-  'Linux')     PLATFORM=linux   ;;
-  'FreeBSD')   PLATFORM=freebsd ;;
-  'WindowsNT') PLATFORM=win     ;;
-  'Darwin')    PLATFORM=mac     ;;
-  'SunOS')     PLATFORM=solaris ;;
-  'AIX')       PLATFORM=linux   ;;
+  'Linux')
+    PLATFORM=linux
+    AUDIO=alsa
+    TARGET_LIST+=,arm-linux-user,i386-linux-user,x86_64-linux-user
+    DISPLAY=sdl
+    PRODUCTS=(bin libexec share)
+    ;;
+  'FreeBSD')
+    PLATFORM=freebsd
+    ;;
+  'WindowsNT')
+    PLATFORM=win
+    ;;
+  'Darwin')
+    PLATFORM=mac
+    AUDIO=coreaudio
+    DISPLAY=cocoa
+    PRODUCTS=(bin share)
```

Comment:
Nope - libexec contains the network bridge helper, which doesn't show up for OSX.